homeassistant/components/honeywell/sensor.py

```diff
@@ -59,9 +59,9 @@ class HoneywellSensorEntityDescription(
 )
 
 
-async def async_setup_entry(hass, config, async_add_entities, discovery_info=None):
+async def async_setup_entry(hass, config_entry, async_add_entities):
```

Comment:
Could you add the type hints here?

```suggestion
async def async_setup_entry(
    hass: HomeAssistant,
    config_entry: ConfigEntry,
    async_add_entities: AddEntitiesCallback,
) -> None:
```

Reply:
Done!